apps/hannk/halide/CMakeLists.txt

```diff
@@ -8,49 +8,49 @@ set(CMAKE_CXX_EXTENSIONS NO)
 
 # ----------------------------
 
-if (NOT CMAKE_CROSSCOMPILING)
+add_custom_target(hannk_tools)
+
+function(_export_hannk_tool TARGET)
+    add_dependencies(hannk_tools ${TARGET})
+    export(TARGETS ${TARGET}
+           NAMESPACE hannk::tools::
+           APPEND FILE "${hannk_BINARY_DIR}/hannk_tools-config.cmake")
+endfunction()
+
+# Emscripten tries to disable finding packages outside its sysroot,
+# but we need our native generators. Setting CMAKE_FIND_ROOT_PATH_BOTH
+# here overrides Emscripten's search preference.
+find_package(hannk_tools QUIET CMAKE_FIND_ROOT_PATH_BOTH)
```

Comment:
AFAIK, no. The argument makes sure the usual search procedure is followed for this call. The only reason we would ever find this package is (1) because the user deliberately wanted to set up a cross-built or (2) someone installed the hannk generators to the system path, but _come on_.

Reply:
ok, no worries then